vignettes/predict_moa_using_snf.Rmd

```diff
@@ -0,0 +1,443 @@
+---
+title: "Predict compounds mechanism-of-action by morphological profiling"
+author: "Allen Goodman and Shantanu Singh"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Predict compounds mechanism-of-action by morphological profiling}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+This vignette demonstrates predicting compound mechanism-of-action using
+morphological profiling data. The data is "cleaned up" using the similarity 
+network fusion algorithm (SNFD). See the vignette `single_cell_analysis` for
+details about this dataset.
+
+```{r libraries, message=FALSE}
+library(dplyr)
+library(magrittr)
+library(ggplot2)
+library(stringr)
+library(cytominergallery)
+library(SNFtool)
+```
+
+# Load data
+
+Per-well profiles computed in `single_cell_analysis` are loaded, as well as
```

Comment:
Refer to the `predict_moa` vignette saying that it is the same data as used there